apps/callscreen/style/oncall.css

```diff
@@ -8,6 +8,7 @@
     font-size: 10px;
     color: white;
     background: transparent;
+    -moz-user-select: none;
```

Comment:
Just to follow up on this one, I removed the html selector and it did not work.  The page was completely distorted.